src/core/tests/type_prop/multinomial.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/multinomial.hpp"
+
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+
+TEST(type_prop, multinomial_i64_i32) {
+    auto probs = std::make_shared<ov::op::v0::Parameter>(ov::element::f64, ov::Shape{4, 4});
+    auto num_shape = ov::op::v0::Constant::create(ov::element::i32, ov::Shape{1}, {4});
+    auto multinomial_func =
+        std::make_shared<ov::op::v13::Multinomial>(probs, num_shape, ov::element::i32, false, false, 0, 0);
```

Comment:
```suggestion
    auto op = make_op(probs, num_shape, ov::element::i32, false, false, 0, 0);
```
If helper test fixture used. Or just rename `multinomial_func`  to `op`, as this is not function.

Reply:
Applied the `make_op` function!